Server/ConcordiaCurriculumManager/Services/DossierService.cs

```diff
@@ -81,6 +82,32 @@ public async Task<Dossier> EditDossier(EditDossierDTO d, User user)
         return dossier;
     }
 
+
+    public async Task<Dossier> DeleteDossier(DeleteDossierDTO d, User user)

```

Comment:
This method does not need to return anything. Unless there is a use case that I am missing.

Reply:
Sounds good